app/uk/gov/hmrc/tariffclassificationfrontend/views/forms/liability_sending_sample_form.scala.html

```diff
@@ -0,0 +1,40 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
+@import uk.gov.hmrc.tariffclassificationfrontend.controllers.routes
+@import uk.gov.hmrc.tariffclassificationfrontend.models.{Case, SampleStatus}
+@import uk.gov.hmrc.tariffclassificationfrontend.views.forms.components.RadioOption
+@import uk.gov.hmrc.tariffclassificationfrontend.views.html.forms.components.input_radiogroup
+@(c: Case, field: Field)(implicit request: Request[_], messages: Messages, appConfig: AppConfig)
+
+
+@helper.form(action = routes.SampleController.updateStatus(c.reference, Some("liability")), 'id -> "sample_status_form") {
+    @helper.CSRF.formField
+    @input_radiogroup(
+        field = field,
+        options = Seq(RadioOption(SampleStatus.AWAITING.toString, "Yes"), RadioOption("", "No")),
```

Comment:
Is your No option not `SampleStatus.NONE`?

Reply:
No.  Yes is Some(Status) (inc. Some(NONE)).   No is None which is achieved via the radio option value "".
I chose AWAITING as a sensible initial sample status when the "is sending sample" state is first turned on.